sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java

```diff
@@ -102,6 +103,7 @@ public HttpLoggingPolicy(HttpLogOptions httpLogOptions) {
                 .map(queryParamName -> queryParamName.toLowerCase(Locale.ROOT))
                 .collect(Collectors.toSet());
             this.prettyPrintBody = false;
+            this.enableRedactedHeaderLogging = true;
```

Comment:
Should we start with `false` as the default and not log redacted headers?

Reply:
this represents the current state and I don't want to change the default behavior

Reply:
ok, I changed the default to false,
i.e. we have a kind-of-a-breaking change from  logging `"foo":"REDACTED","bar":"REDACTED"` to:
- not logging it by default
- enabling it back with flag on the options to appear in the "redactedHeaders:"foo,bar".  Since this alone is somewhat breaking, I'd rather change the default to not logging them,

LMK what you think about breaking-ish change and I'm fine wither way.

Reply:
updated default back to enabled